conf/repo/oca.yml

```diff
@@ -54,3 +54,10 @@ oca-weblate-deployment:
   name: Oca Weblate Deployment
   psc: community-maintainers
   psc_rep: community-maintainers-psc-representative
+docs:
+  branches: []
+  category: Oca
+  default_branch: master
+  name: The documentation of OCA module
+  psc: board
+  psc_rep: board
```

Comment:
Is there a way to add individual maintainers/admins here?

Reply:
I think that's an undocumented feature. You should be able to use `maintainers` https://github.com/OCA/repo-maintainer/blob/master/oca_repo_maintainer/tools/manager.py#L166

Reply:
At the moment @yajo needs admin rights, so I'll leave this draft for now.